recipes/qdarkstyle/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "qdarkstyle" %}
+{% set package_name = "QDarkStyle" %}
+{% set version = "2.5.4" %}
+{% set hash_type = "sha256" %}
+{% set hash = "3eb60922b8c4d9cedecb6897ca4c9f8a259d81bdefe5791976ccdf12432de1f0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package_name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
```

Comment:
This project is a pure python project, right? So, please add ``noarch: python`` and also add ``pip`` as a ``host`` dependency

Reply:
Yes, I was about to suggest that too.